app/signup.tsx

```diff
@@ -64,6 +67,9 @@ export default function Signup() {
         </Text>
         <TextInput
           secureTextEntry
```

Comment:
Slightly bing that `secureTextEntry` is before in this block, but after in the other password field.